roxie/ccd/ccdfile.cpp

```diff
@@ -860,9 +860,9 @@ class IndexCacheWarmer : implements ICacheWarmer
 class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress, public CInterface
 {
     friend class CcdFileTest;
-    mutable ICopyArrayOf<ILazyFileIO> todo; // Might prefer a queue but probably doesn't really matter.
+    mutable IArrayOf<ILazyFileIO> todo; // Might prefer a queue but probably doesn't really matter.
 #ifdef _CONTAINERIZED
-    mutable ICopyArrayOf<ILazyFileIO> buddyCopying;
+    mutable IArrayOf<ILazyFileIO> buddyCopying;
```

Comment:
checkBuddies is an ICopyArrayOf and we might do a swapWith() between buddyCopying and checkBuddies, is that always ok ?